test/e2e/util.go

```diff
@@ -1721,6 +1722,14 @@ func getNodeEvents(c *client.Client, nodeName string) []api.Event {
 	return events.Items
 }
 
+func ListSchedulabeNodesOrDie(c *client.Client) *api.NodeList {
```

Comment:
We may, but then we can write List directly. It's a helper method for the common use case, which in addition makes code slightly more readable by removing this error check.